apps/evi/agent/instructions/memory.ts

```
@@ -0,0 +1,26 @@
+ * resolving it per turn would invalidate everything behind it.
+ */
+export default defineDynamic({
+  events: {
+    'session.started': async (_event, ctx) => {
+      if (!memoryAvailable()) return null
+      try {
+        const session = await openMemorySession(ctx.session.auth.current)
+        if (session === null) return null
+        const markdown = await buildCoreBlock(session)
+        return markdown === null ? null : defineInstructions({ markdown })
+      }
```

ESLint

[error] Closing curly brace does not appear on the same line as the subsequent block.

(brace-style)